README.md

```diff
@@ -2,17 +2,30 @@
 
 This is a minimal Python package that allows users to download, query, and upload data from the IMAP Science Data Center (SDC).
 
-## Configuration
+## Command Line Utility
 
-The folder structure for data files within the IMAP SDC is rigidly
-defined, so the data access will mimic that structure to make sure
-all data is stored in the same heirarchical structure as the SDC.
-This will enable seamless transition between a user's local system
-and the SDC.
+### Query / Search for data
 
-A user's root data location can be specified as an environment
-variable ``IMAP_DATA_DIR`` or through a configuration dictionary
-within the package itself ``imap_data_access.config["DATA_DIR"]``.
+```bash
+# Find all files from the SWE instrument
+imap-data-access query --instrument swe
+# Find all files during the year 2025
+imap-data-access query --start_date 20250101 --end_date 20251231
```

Comment:
We might want to use parameters here that actually return results, otherwise users might get discouraged or think something is wrong if they get an empty result?

Reply:
Good suggestion, I updated to 2024